app/javascript/components/public_health/CustomExport.js

```diff
@@ -214,46 +214,90 @@ class CustomExport extends React.Component {
                     checked={this.state.selected_records === 'current'}
                   />
                   {this.state.selected_records === 'current' && (
-                    <div className="px-1 pb-1">
-                      <Badge variant="secondary" className="mr-1">
-                        {`${_.capitalize(this.props.patient_query.workflow)} - ${this.props.tabs[this.props.patient_query.tab]?.label}`}
-                      </Badge>
+                    <div className="custom-export-filters">
+                      <p className="custom-export-filter-text">
```

Comment:
does this need to be a p tag? or can it be a span?  I'm getting a console error about nested p tags

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/980/commits/4cdc5386acf1f2d49a3076e5b48cda1354c3e0f6#diff-14793b3bc71143b45317a3d08474d370db9ffdfb9643e754341eeda1ab5f096bR218